phoenix-pherf/src/test/java/org/apache/phoenix/pherf/PherfTest.java

```diff
@@ -40,4 +44,48 @@ public void testUnknownOption() {
         exit.expectSystemExitWithStatus(1);
         Pherf.main(args);
     }
+
+    @Test
+    public void testDefaultLogPerNRowsArgument() throws Exception {
+        String[] args = {"-listFiles"};
+        Pherf pherf = new Pherf(args);
+        Properties prop = pherf.getProperties();
+        assertEquals(Long.valueOf(PherfConstants.LOG_PER_NROWS),
```

Comment:
Having a constant called LOG_PER_NROWS and another called LOG_PER_N_ROWS in the same class is pretty confusing, especially when one refers to a key and the other to a value. :-)

Reply:
Will change `LOG_PER_N_ROWS` to `LOG_PER_NROWS_NAME`